scala/private/macros/scala_repositories.bzl

```diff
@@ -199,3 +199,18 @@ def scala_repositories(
         name = "io_bazel_rules_scala/dependency/scala/guava",
         actual = "@io_bazel_rules_scala_guava",
     )
+
+    native.bind(
+        name = "io_bazel_rules_scala/dependency/scalac_rules_commons_io",
```

Comment:
What is the motivation behind having a second bind for `@scalac_rules_commons_io`?

Nit: name does not follow the same pattern as others binds.

Reply:
> second bind

This isn't a second `bind`, right? `@scalac_rules_commons_io` is a repo. `Bind`s end up in `external`? Is this what you're commenting on?

My thinking is in [ExternalRepositories](https://github.com/smparkes/rules_scala/blob/configuration/unstable/multiscala/ExternalRepositories.md). The tl;dr is that using repo names makes things loader(e.g., `jvm_import_external`, `maven_install`)-specific: different loaders create different repo names and the loaders don't make this configurable. There are a few ways to address this, including using `bind`s to map from loader-specific repo names to common bind targets.

If this approach is acceptable, it seems prudent to come up with a canonical mapping from maven coordinate to bind target. At this point, I didn't want to presume that conclusion so I was sticking with minimal changes necessary to unblock other work. I took the approach of using the existing repo name as the bind target.

Reply:
I am referring to this one
```
native.bind(
        name = "io_bazel_rules_scala/dependency/commons_io/commons_io",
        actual = "@scalac_rules_commons_io//jar",
)
```
on this line: https://github.com/bazelbuild/rules_scala/blob/0ab261f60ac82ee407a8457f84b94132ca808173/scala/private/macros/scala_repositories.bzl#L165

Reply:
Ah. Thanks. I was pushing too fast and didn't look for the bind: it wasn't being used anywhere within `rules_scala` itself.
Removed the redundant bind and updated the scalac build to use it.